### ISSUE TITLE
New access tag: "ambiguous"

### ISSUE THREAD

**Issue by pmonks** [theCrag/website#3429]
It would be helpful to have a new access tag called something like "ambiguous", which ideally triggers the display of some kind of header text (similar to how the closed tag works, albeit with different text).

That text might say something like:
```
The access status of climbing in this area is ambiguous.  You are encouraged to climb elsewhere.
```

**Comment by brendanheywood** [comment]
Note we already have these, at this stage only closed gets special treatment. I'm thinking Illegal could be removed and just converted to Closed.

Designated 
 Allowed 
 Private
 Tolerated 
Illegal
 Closed 
Paid

https://www.thecrag.com/en/article/tagging#access-and-approach

**Comment by pmonks** [comment]
Yeah `illegal` and `closed` seem duplicative to me, however I don't see anything in that list that would be equivalent to `ambiguous`.

**Comment by rouletout** [comment]
So is this the same as "At Risk" then? See https://github.com/theCrag/website/issues/3435

**Comment by Mdemaillard** [comment]
-1 Anything which is not clear cut will need explanation, so better fit in a Warning and or access description field. I would leave the tags for clear cut status.
Once the ambiguity (and warning) is resolved, a tag can be set.

+1 to merge illegal into closed.
